### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by alanyee**
-Add Python formatting for TF website
-Refactor comment
-Import statements are shorter than 80 chars, so pylint should not need to be suppressed here
-Add python formatting to correct website formatting away newlines

### CODE REVIEW

File: tensorflow/contrib/deprecated/__init__.py [hunk 1]
```diff
@@ -18,42 +18,40 @@
...
-
 tf.merge_summary -> tf.summary.merge
-
 tf.merge_all_summaries -> tf.summary.merge_all
+```
 
-We think this is a cleaner API and will improve long-term discoverability and
-clarity of the TensorFlow API. However, we also took the opportunity to make an
+We think this API is cleaner and will improve long-term discoverability and
+clarity of the TensorFlow API. We however, also took the opportunity to make an
```
Comment (martinwicke) [inline tensorflow/contrib/deprecated/__init__.py]: You either need another comma before 'however', or you revert this line. I'd go with the old order.

File: tensorflow/contrib/deprecated/__init__.py [hunk 2]
```diff
@@ -18,42 +18,40 @@
...
 generated protobufs.
 
-Previously, the tag was allowed to be any unique string, and had no relation
+Previously, the tag was allowed to be any unique string; it had no relation
 to the summary op generating it, and no relation to the TensorFlow name system.
-This made it very difficult to write re-usable code that would add summary
-ops to the graph. If you had a function that would add summary ops, you would
-need to manually pass in a name scope to that function to create deduplicated
-tags, otherwise your program would fail with a runtime error due to tag
+This behavior greatly complicates the reusability of the code that would add
```
Comment (martinwicke) [inline tensorflow/contrib/deprecated/__init__.py]: reusability cannot be made more complicated. Reusing things can be. I think the old phrasing is better.

File: tensorflow/contrib/deprecated/__init__.py [hunk 3]
```diff
@@ -18,42 +18,40 @@
...
-Previously, the tag was allowed to be any unique string, and had no relation
+Previously, the tag was allowed to be any unique string; it had no relation
 to the summary op generating it, and no relation to the TensorFlow name system.
-This made it very difficult to write re-usable code that would add summary
-ops to the graph. If you had a function that would add summary ops, you would
-need to manually pass in a name scope to that function to create deduplicated
-tags, otherwise your program would fail with a runtime error due to tag
+This behavior greatly complicates the reusability of the code that would add 
+summary ops to the graph. If you had a function to add summary ops, you would
+need to pass in a namescope, manually, to that function to create deduplicated
```
Comment (martinwicke) [inline tensorflow/contrib/deprecated/__init__.py]: if you want to make the nature of "name scope" clearer, use `name_scope`.

Comment (martinwicke) [reply tensorflow/contrib/deprecated/__init__.py]: Or even better: `tf.name_scope`.

**Review by martinwicke — CHANGES_REQUESTED** [review]

**Comment by alanyee** [comment]
@martinwicke does the commit look good now?

**Comment by drpngx** [comment]
Jenkins, test this please.

@martinwicke any thoughts on the new version?

**Comment by alanyee** [comment]
@drpngx @martinwicke any thoughts on the changes?

**Review by martinwicke — APPROVED** [review]

**Comment by martinwicke** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
Hitting the permission denied. @gunan FYI it looks like we are seeing more of [this](https://ci.tensorflow.org/job/tensorflow-pull-requests-makefile/10811/) recently.

Jenkins, test this please.

**Pull request merged**